docs/development.md

```diff
@@ -21,43 +21,48 @@ For detailed information about `craig-stars` architecture, check out the [archit
 
 # Getting Started
-**Note**: The following instructions assume you have `make`, `go`, and `npm` installled.
+**Note**: The following instructions assume you have `go`, `make` and `npm` installled. `go` can be installed quite easily from [their website](https://go.dev/dl/]), while the easiest way to install the other 2 is to use a package manager: [chocolatey](https://chocolatey.org/install) for Windows, [Homebrew](https://brew.sh/) or apt-get/yum for Linux (& Chromebook by extension).
 
-After cloning the repo, build `craig-stars` locally once. This will build the frontend and then the backend. The backend embeds the frontend resources into its binary to support a single binary deployment. 
+From there, you can install `make` and `node.js` (which includes npm) fairly easily. NOTE: If `node.js` doesn't co-operate after being installed, try uninstalling it and re-installing the **LTS** version from their website instead.
 
-## Assets
-You will need art assets, and those can be downloaded from [https://craig-stars.net/images/images.zip](https://craig-stars.net/images/images.zip). Copy these images into the `frontend/static/images` folder.
+After cloning the repo (easily done with Github Desktop), build `craig-stars` locally once. This will build the frontend and then the backend. The backend embeds the frontend resources into its binary to support a single binary deployment. 
 
-## Building and Running
+### Installing go-sqlite3
+If you don't already have it, you'll also need to install [go-sqlite3](https://github.com/mattn/go-sqlite3) for things to work. However, this itself requires the [GCC compiler collection](https://gcc.gnu.org/) to function. 
 
-```bash
-make build
-```
+For Linux/Mac, you can simply install the latest version of the GCC compilers using whatever package software you installed previously.
 
-Install air for automatic server restarts while developing: 
+However, on Windows, you'll need to install a Linux-like development interface (use [MinGW](https://www.mingw-w64.org/) - Cygwin64 WILL NOT WORK) to install & build the latest GCC version.
 
-```bash
-go install github.com/air-verse/air@latest
-```
+Once that's done, enter `go install github.com/mattn/gosqlite3` into your computer's native terminal (or the terminal in VS Code) to install the driver.
+
+## Assets
+You will need art assets, and those can be downloaded from [https://craig-stars.net/images/images.zip](https://craig-stars.net/images/images.zip). Copy these images into the `frontend/static/images` folder (or make a new folder if it doesn't already exist).
+
+## Building and Running
+Finally, go to your terminal and enter the following commands:
 
-Launch the frontend and backend at the same time with `make`
 ```bash
+go install github.com/air-verse/air@latest
+make build
 make dev
+go run main.go serve --debug
 ```
 
+(Installing air is only required once, so you can skip the first line on subsequent builds.)
```

Comment:
Let's add a separate section for `air`. Users can just run `make dev` until they are making frequent changes to the backend.